packages/shared/ReactTypes.js

```diff
@@ -178,18 +178,29 @@ export type Awaited<T> = T extends null | void
     : T // argument was not an object
   : T; // non-thenable
 
+export type ReactCallSite = [
+  string, // function name
+  string, // file name TODO: model nested eval locations as nested arrays
```

Comment:
👀

Reply:
I figured the model could maybe be:

```
export type ReactCallSite = [
  string, // function name
  string | ReactCallSite, // file name or eval
]
```

Where if it's a nested callsite that implies it's eval at that location.